packages/sources/coinmarketcap-test/src/endpoint/crypto.ts

```diff
@@ -141,33 +141,36 @@ const httpTransport = new HttpTransport<CryptoEndpointTypes>({
     }
 
     for (const [idType, fullList] of Object.entries(groupedParams)) {
-      // CMC does not support more than 120 unique quotes
-      const chunkedList = chunkArray(fullList, 120)
-
-      // This could be further optimized in cases with more than 120 entries, to make sure that
-      // chunkes are grouped optimally to avoid sending unnecessary converts
-      for (const list of chunkedList) {
-        requests.push({
-          params: list,
-          request: {
-            baseURL: settings.API_ENDPOINT,
-            url: '/cryptocurrency/quotes/latest',
-            headers: {
-              'X-CMC_PRO_API_KEY': settings.API_KEY,
+      if (fullList && fullList.length > 0) {
```

Comment:
I prefer continues over nested ifs, or a filter even, but it's still ok and I wanna push forward. Just leaving as a comment